pkg/rke/templates/templates.go

```diff
@@ -252,10 +254,18 @@ func LoadK8sVersionedTemplates() map[string]map[string]string {
 			">=1.25.9-rancher2-1 <1.25.9-rancher2-2":   aciv5236,
 			">=1.25.9-rancher2-2 <1.25.13-rancher1-1":  aciv5271,
 			">=1.25.13-rancher1-1 <1.25.16-rancher2-2": aciv6031,
-			">=1.25.16-rancher2-2 <1.26.0-rancher0":    aciv6032,
+			">=1.25.16-rancher2-2 <1.25.16-rancher2-3": aciv6032,
+			">=1.25.16-rancher2-3 <1.26.0-rancher0":    aciv6033,
 			">=1.26.0-rancher0 <1.26.8-rancher1-1":     aciv5271,
 			">=1.26.8-rancher1-1 <1.26.11-rancher2-2":  aciv6031,
-			">=1.26.11-rancher2-2":                     aciv6032,
+			">=1.26.11-rancher2-2 <1.26.11-rancher2-3": aciv6032,
+			">=1.26.11-rancher2-3 <1.26.14-rancher1-1": aciv6033,
+			">=1.26.14-rancher1-1 <1.27.0-rancher1-1":  aciv6041,
+			">=1.27.0-rancher1-1 <1.27.8-rancher2-1":   aciv5271,
+			">=1.27.8-rancher2-1 <1.27.8-rancher2-2":   aciv6031,
+			">=1.27.8-rancher2-2 <1.27.8-rancher2-3":   aciv6032,
+			">=1.27.8-rancher2-3 <1.27.11-rancher1-1":  aciv6033,
```

Comment:
Same comment as above but for `v1.27.10-rancher1-1` version.

Reply:
Done